### ISSUE TITLE
Domain adaptation in TFBS section

### ISSUE THREAD

**Issue by agitter** [greenelab/deep-review#728]
We merged #532 in order to clear all open pull requests for a end-to-end Markdown reflow.  This is a reminder that @jacklanchantin or @akundaje planned to make additional edits to the TF binding section.

We discussed adding FactorNet (#549), @jacklanchantin's transfer string kernel, the DREAM challenge, and other related work here.

I'll make an update here when we're ready for a new pull request.

**Comment by agitter** [comment]
The reflow is done so we're ready for a new pull request to wrap this up.  We now have one sentence per line throughout the manuscript.

Be sure to pull the latest master before starting the new pull request.
